### PR TITLE
feat(config): add Aeotec illumino switches

### LINKED ISSUE #3386: Missing device configuration: Aeotec Illumino Wall Switch
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

Aeotec Illumino Wall Switch

### Manufacturer ID

0x371

### Product Type

0x103

### Product ID

0x26

### Firmware Version

1.4

### Is the device listed on the Z-Wave Alliance website?

https://products.z-wavealliance.org/products/4273

### Do you have a manual?

https://help.aeotec.com/support/solutions/articles/6000247459-illumino-wall-switch-user-guide

### LINKED ISSUE #3388: Aeotec Dual Nano Switch ZW132: Wall Swipe Button Brightness can be 255
https://devices.zwave-js.io/?jumpTo=0x0086:0x0003:0x0084:0.0
```
zwave2.0
2021-09-22 13:38:59.736 warn State value to set for "zwave2.0.Node_015.Configuration.ledIndicatorOnRightGestureBrightness" has value "255" greater than max "99"
zwave2.0
2021-09-22 13:38:59.586 warn State value to set for "zwave2.0.Node_015.Configuration.ledIndicatorOnLeftGestureBrightness" has value "255" greater than max "99"
zwave2.0
2021-09-22 13:38:59.469 warn State value to set for "zwave2.0.Node_015.Configuration.ledIndicatorOnDownGestureBrightness" has value "255" greater than max "99"
zwave2.0
2021-09-22 13:38:59.412 warn State value to set for "zwave2.0.Node_015.Configuration.ledIndicatorOnUpGestureBrightness" has value "255" greater than max "99"
zwave2.0
2021-09-22 13:38:59.314 warn State value to set for "zwave2.0.Node_015.Configuration.wallswipeButtonBrightness" has value "255" greater than max "99"
```

### PR DESCRIPTION
**Pull request by blhoward2**
Closes #3386
Closes #3388

### CODE REVIEW

File: packages/config/config/devices/0x0086/templates/aeotec_template.json [hunk 1]
```diff
@@ -1557,6 +1634,56 @@
...
+			{
+				"label": "Switch; 2x switch and dim to param 132/133; 3x switch and begin dimmer ramp",
+				"value": 1
+			},
+			{
+				"label": "Momentary; 2x momentary and dim to param 132/133 (hold to dim up/down)",
+				"value": 2
+			},
+			{
+				"label": "Momentary; 2x momentary and dim to param 132/133 (decreasing dim on hold)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0086/templates/aeotec_template.json]: I had to read this a couple of times and I'm still not sure I understand it correctly. What about this?
* Switch - Double Click to dim to param 132/133; Triple Click to dim up (?)
* Momentary - Double Click and Hold to dim up/down to param 132/133
* Momentary - Double Click and Hold to dim down to param 132/133

File: packages/config/config/devices/0x0371/zwa037.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Top Button - Basic Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Top Button - MultiLevel Set",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa037.json]:
```suggestion
			"label": "Top Button - Multilevel Set",
```
carry throughout

File: packages/config/config/devices/templates/master_template.json [hunk 3]
```diff
@@ -403,6 +426,33 @@
...
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Local control only",
+				"value": 1
+			},
+			{
+				"label": "Central scene only",
```
Comment (AlCalzone) [inline packages/config/config/devices/templates/master_template.json]:
```suggestion
				"label": "Central Scene only",
```
This is a CC name, so we should capitalize it.

File: packages/config/config/devices/templates/master_template.json [hunk 4]
```diff
@@ -403,6 +426,33 @@
...
+				"value": 2
+			},
+			{
+				"label": "Local and central scene",
```
Comment (AlCalzone) [inline packages/config/config/devices/templates/master_template.json]:
```suggestion
				"label": "Local and Central Scene",
```
dito

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0086/templates/aeotec_template.json (hunk 1 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0086/templates/aeotec_template.json]: I'm not sure I understand either, truthfully. Its a mess. I think this is right as I’ve edited it.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**